client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -36,8 +34,15 @@ class TwoFactorActions extends Component {
 
 		this.props.sendSmsCode( userId, twoStepNonce ).then( () => {
 			page( login( { twoFactorAuthType: 'sms' } ) );
+			this.props.setNotice( {
+				message: 'Sent SMS successfully',
```

Comment:
We'll add a nice message for this in https://github.com/Automattic/wp-calypso/pull/14100